packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -165,6 +168,15 @@ public ResponseEntity<List<Project>> getProjects(
 						counts.getOrDefault(AssetType.PUBLICATION, 0).toString());
 
 				project.setMetadata(metadata);
+
+				// Set the creator name for the project
+				if (project.getUserId() != null) {
+					final String creatorName =
+							userService.getById(project.getUserId()).getName();
+					if (creatorName != null) {
+						project.setUserName(creatorName);
```

Comment:
Will this save the new author in the database, or it will be done everytime?

Reply:
authors are not saved in the database